samples/istio/quickstart/experiment.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: iter8.tools/v2alpha2
+kind: Experiment
+metadata:
+  name: istio-quickstart
+spec:
+  # target identifies the service under experimentation using its fully qualified name
+  target: bookinfo-iter8/productpage
+  strategy:
+    # this experiment will perform an A/B test
+    testingPattern: A/B
+    # this experiment will progressively shift traffic to the winning version
+    deploymentPattern: Progressive
+    actions:
+      # when the experiment completes, promote the winning version using kubectl apply
+      finish:
+      - task: common/exec
+        with:
+          cmd: /bin/bash
+          args: 
+          - "-c"
+          - kubectl -n {{ .namespace }} apply -f {{ .promote }}
+
+  criteria:
+    rewards: # metrics to be used to determine the "value" or "benefit" of a version
+    - metric: books-purchased
+      preferredDirection: High
+    objectives: # metrics to be used to determine validity of a version
+    - metric: iter8-istio/mean-latency
+      upperLimit: 100
+    - metric: iter8-istio/error-rate
+      upperLimit: "0.01"
+  duration: # product of fields determines length of the experiment
+    intervalSeconds: 10
+    iterationsPerLoop: 10
+  versionInfo:
+    # information about the app versions used in this experiment
+    baseline:
+      name: A
+      variables:
+      - name: revision # used in Prometheus queries
```

Comment:
`revision` is more of a Knative terminology. Perhaps `subset` or `deployment` is a better term for Istio?

Reply:
Making this change has an impact on the metrics definitions. This works for both deployments and services. I would prefer a word that is neutral. I'll use "version".